scripts/advisor/run_advisor.py

```diff
@@ -111,16 +96,17 @@ def run_with_advisor(path, output, name, exec_args, advisor_home, plot):
     ]
     advisor_survey = [
         '-collect survey',
-        '-start-paused',
         '-run-pass-thru=--no-altstack',  # Avoids `https://software.intel.com/en-us/vtune-amplifier-help-error-message-stack-size-is-too-small`  # noqa
         '-strategy ldconfig:notrace:notrace',  # Avoids `https://software.intel.com/en-us/forums/intel-vtune-amplifier-xe/topic/779309`  # noqa
         '-start-paused',  # The generated code will enable/disable Advisor on a loop basis
     ]
     advisor_flops = [
         '-collect tripcounts',
+        '-enable-cache-simulation',
         '-flop',
+        '-start-paused',
     ]
-    py_cmd = ['python', str(path)] + exec_args.split()
+    py_cmd = ['python3', str(path)] + exec_args.split()
```

Comment:
this assumes something about the underneath installation

perhaps you may use this: https://github.com/devitocodes/devito/blob/master/tests/conftest.py#L129
?